packages/lambda-xyz/src/wmts.capability.ts

```diff
@@ -22,12 +24,12 @@ for (const k of ImageFormatOrder) {
     formats.push(V('Format', 'image/' + k));
 }
 
-const tileMatrixSetId: Partial<Record<TileSetType, string>> = {
-    [TileSetType.aerial]: 'GoogleMapsCompatible',
+const tileMatrixSetId: Partial<Record<string, string>> = {
+    aerial: 'GoogleMapsCompatible',
 };
 
-const LayerPreamble: Partial<Record<TileSetType, VNodeElement[]>> = {
-    [TileSetType.aerial]: [
+const LayerPreamble: Partial<Record<string, VNodeElement[]>> = {
+    aerial: [
         V('ows:Title', 'NZ Aerial Imagery Basemap'),
```

Comment:
I think we need to move this into the database.

Reply:
Do you want to do it now?  That is a bit of planning etc. We can make another ticket for that work?

Reply:
Ok lets make the database changes for the next PR